test/browser/test.js

```diff
@@ -146,6 +146,13 @@ function runTest(name, autoTest) {
 
```

Comment:
Could you give an advice where I should add such test? I have not found tests for such callbacks in the suite.